src/common/interop/PowerToys.Interop.vcxproj.filters

```diff
@@ -68,6 +68,9 @@
       <Filter>Resource Files</Filter>
     </ResourceCompile>
   </ItemGroup>
+  <ItemGroup>
+    <Natvis Include="$(MSBuildThisFileDirectory)..\..\natvis\wil.natvis" />
```

Comment:
Always wonder why these get added automatically through the solution, but looks like there's no issue in adding it.